### ISSUE TITLE
Environment

### ISSUE THREAD

**Issue by zhangcong30** [Yzichen/PolarBEVDet#7]
Excuse me, when I install all packages in conda and run 'python tools/create_data_bevdet.py', the error comes 

<img width="1369" height="609" alt="Image" src="https://github.com/user-attachments/assets/c6daab96-8657-4852-9606-cba5364f8112" />

my pip list is like follows
(polarbevdet) [zhangcong@G101 PolarBEVDet]$ pip list
Package                     Version      Editable project location
--------------------------- ------------ ----------------------------------------------------------
absl-py                     0.15.0
addict                      2.4.0
aliyun-python-sdk-core      2.16.0
aliyun-python-sdk-kms       2.16.5
array_record                0.5.1
astunparse                  1.6.3
black                       25.11.0
cachetools                  6.2.4
certifi                     2026.1.4
cffi                        2.0.0
charset-normalizer          3.4.4
clang                       5.0
click                       8.1.8
colorama                    0.4.6
contourpy                   1.3.0
crcmod                      1.7
cryptography                46.0.3
cycler                      0.12.1
decorator                   5.2.1
descartes                   1.1.0
dm-tree                     0.1.8
etils                       1.5.2
exceptiongroup              1.3.1
filelock                    3.14.0
fire                        0.7.1
flake8                      7.3.0
flatbuffers                 1.12
fonttools                   4.60.2
fsspec                      2025.10.0
gast                        0.4.0
google-pasta                0.2.0
grpcio                      1.74.0
h5py                        3.1.0
idna                        3.11
ImageIO                     2.37.2
immutabledict               4.2.2
importlib_metadata          8.7.1
importlib_resources         6.5.2
iniconfig                   2.1.0
jmespath                    0.10.0
joblib                      1.5.3
keras                       2.15.0
Keras-Preprocessing         1.1.2
kiwisolver                  1.4.7
llvmlite                    0.36.0
lyft-dataset-sdk            0.0.8
Markdown                    3.9
markdown-it-py              3.0.0
MarkupSafe                  3.0.3
matplotlib                  3.5.3
mccabe                      0.7.0
mdurl                       0.1.2
mmcls                       0.25.0
mmcv-full                   1.6.0
mmdet                       2.28.2
mmdet3d                     1.0.0rc6
mmsegmentation              0.30.0
model-index                 0.1.11
mypy_extensions             1.1.0
narwhals                    2.15.0
networkx                    2.2
ninja                       1.13.0
numba                       0.53.0
numpy                       1.23.5
nuscenes-devkit             1.1.11
opencv-python-headless      4.13.0.90
opendatalab                 0.0.10
OpenEXR                     3.4.4
openmim                     0.3.9
openxlab                    0.1.3
opt-einsum                  3.3.0
ordered-set                 4.1.0
oss2                        2.17.0
packaging                   24.2
pandas                      2.3.3
pathspec                    1.0.3
Pillow                      8.4.0
pip                         25.3
platformdirs                4.4.0
plotly                      6.5.2
pluggy                      1.6.0
plyfile                     1.1.3
polarbevdet-plugin          0.0.0        /gdata2/zhangcong/workspase_2026/code/PolarBEVDet/projects
prettytable                 3.16.0
promise                     2.3
protobuf                    6.32.0
psutil                      7.2.1
pycocotools                 2.0.11
pycodestyle                 2.14.0
pycparser                   2.23
pycryptodome                3.23.0
pyflakes                    3.4.0
Pygments                    2.19.2
pyparsing                   3.3.1
pyquaternion                0.9.9
pytest                      8.4.2
python-dateutil             2.9.0.post0
pytokens                    0.3.0
pytz                        2023.4
PyWavelets                  1.6.0
PyYAML                      6.0.3
requests                    2.28.2
rich                        13.4.2
scikit-image                0.19.3
scikit-learn                1.6.1
scipy                       1.10.1
setuptools                  59.5.0
Shapely                     1.8.5.post1
six                         1.15.0
tabulate                    0.9.0
tensorboard                 2.20.0
tensorboard-data-server     0.7.2
tensorflow                  2.6.0
tensorflow-addons           0.23.0
tensorflow-datasets         4.9.3
tensorflow-estimator        2.15.0
tensorflow-graphics         2021.12.3
tensorflow-metadata         1.17.3
termcolor                   1.1.0
terminaltables              3.1.10
threadpoolctl               3.6.0
tifffile                    2024.8.30
toml                        0.10.2
tomli                       2.4.0
torch                       1.13.1+cu117
torchaudio                  0.13.1+cu117
torchvision                 0.14.1+cu117
tqdm                        4.65.2
trimesh                     4.2.0
typeguard                   2.13.3
typing-extensions           3.7.4.3
tzdata                      2025.3
urllib3                     1.26.20
waymo-open-dataset-tf-2-6-0 1.4.8
wcwidth                     0.2.14
Werkzeug                    3.1.5
wheel                       0.45.1
wrapt                       1.12.1
yapf                        0.43.0
zipp                        3.23.0
So, how can i fix it ?
